### ISSUE TITLE
More parameters are required to reproduce the experiment

### ISSUE THREAD

**Issue by 6syun9** [m3dev/gokart#72]
Hi. @nishiba @hirosassa 

Now, we are writing the parameter log in `~/task_param/*` as follows.
```
 {"from_date": "2019-07-04", "to_date": "2019-07-05", "type": "hoge"}
```
But, only its own parameters ,so other parameters are required to reproduce the experiment.  
We also need the parameters of the task being `luigi.task.flatten(self.requires())`.

Ideal for example...
```
 {"from_date": "2019-07-04", "to_date": "2019-07-05", "type": "hoge", "interval": 1, "item_type": "piyo", ...}
```

We should edit here!
https://github.com/m3dev/gokart/blob/master/gokart/task.py#L247

**Comment by 6syun9** [comment]
From #73 
> For example, I want to get the string data/piyo.csv from outside of gokart in following cases,.
It's difficult because we need to write self.make_target('data/piyo.csv') directly to the code, and make_target need require's parameter.
```
class Hoge(gokart.TaskOnKart)
    def output(self):
        return self.make_target('data/piyo.csv')
```

I want get `data/piyo.csv`...

**Comment by nishiba** [comment]
When we know the task id, please give the id  to the 'make_target' as follow:
`self.make_target('data/piyo_THE_TASK_IS.csv', use_unique_id=False)`

**Comment by 6syun9** [comment]
@nishiba We can't get string (`data/piyo_{}_.csv`) from modulized `Hoge`.
We can see that if you look at the library source, but we usually don't want it...

**Comment by 6syun9** [comment]
fix #76
